tests/sanity/ignore-2.10.txt

```diff
@@ -9,6 +9,7 @@ plugins/modules/acme_account_info.py validate-modules:return-syntax-error
 plugins/modules/acme_challenge_cert_helper.py validate-modules:return-syntax-error
 plugins/modules/ecs_certificate.py validate-modules:invalid-documentation
 plugins/modules/get_certificate.py validate-modules:invalid-documentation
+plugins/modules/luks_device.py validate-modules:invalid-documentation
```

Comment:
curiousity, what tripped the test? `text` as a value?

Reply:
Nope, `choices` as a dictionary in `DOCUMENTATION`.

Reply:
Why is that not allowed? It makes the docs better!